src/NITTA/Model/Networks/Bus.hs

```diff
@@ -71,6 +71,8 @@ data BusNetwork tag v x t = BusNetwork
       bnSignalBusWidth :: Int
     , ioSync :: IOSynchronization
     , bnEnv :: UnitEnv (BusNetwork tag v x t)
+    , -- |Bus control signal tags available for binding
+      bnAvailPorts :: [SignalTag]
```

Comment:
I think you do not need to store it here because you can reallocate than from scratch on each network modification. 

At the same time, the comment doesn’t allow to understands the meaning of that field.